pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
 
    <groupId>ome</groupId>
-   <version>5.2.0</version>
```

Comment:
The benchmarking code will go into the performance repo alongside the ome-files and bio-formats tests.  The version bump is all I have planned for this repo.